R/derive_joined.R

```diff
@@ -806,17 +806,92 @@ get_joined_data <- function(dataset,
 
   # join the input dataset with itself such that to each observation of the
   # input dataset all following observations are joined
+  data_add_to_join <- select(
```

Comment:
is it possible to have a test that fires periodically to check this is working as intended.  

Perhaps we could use https://testthat.r-lib.org/reference/skip.html `skip_on_cran` so it doesn't fire on CRAN servers...but maybe every now and then a CI job could fire that runs this test on GitHub?

Reply:
I'm not sure what you mean. What should be tested periodically?

Reply:
I see. It would be interesting to test on performance issues (run time and memory consumption) but I think this requires more discussion on how to set it up. And then it would require time to implement it.

Reply:
added to discussion